### PR TITLE
[fix][DTT-126]: fix(legacy): download of legacy

### PR DESCRIPTION
**Pull request by anmaxvl**

### CODE REVIEW

File: gdc_client/query/versions.py [hunk 1]
```diff
@@ -31,13 +31,13 @@ def get_latest_versions(url, uuids):
...
         for result in resp.json():
-            # Probably wouldn't happen, but just in case add an error check.
             file_id = result.get("id")
             uuid = result.get('latest_id')
             if uuid:
                 latest_versions[file_id] = uuid
                 continue
-            logger.info("No latest version found for {}".format(file_id))
+            # Might happen for legacy files
+            latest_versions[file_id] = file_id
```
Comment (ProfOak) [inline gdc_client/query/versions.py]: Tthis overwrites the previous block's results. You need to extend the previous block with an else case.

```python
if uuid:
    latest_versions[file_id] = uuid
else:
    latest_versions[file_id] = file_id
```

**Review by ProfOak — CHANGES_REQUESTED** [review]

File: gdc_client/query/versions.py (hunk 1 above)
Comment (anmaxvl) [reply gdc_client/query/versions.py]: it won't, since there is a `continue` statement.

Comment (anmaxvl) [reply gdc_client/query/versions.py]: I added unit tests for it last time as well.

Comment (ProfOak) [reply gdc_client/query/versions.py]: Oh I see, sorry I misread. You could rewrite this without a continue though.

Comment (anmaxvl) [reply gdc_client/query/versions.py]: yeah, sure

**Review by ProfOak — APPROVED** [review]
:+1:

**Pull request merged**